doc/source/parameters/dem/insertion_info.rst

```diff
@@ -10,7 +10,7 @@ In this subsection, insertion methods which are ``volume``, ``plane``, ``list``
 .. code-block:: text
 
   subsection insertion info
-    # Choices are volume|plane|list|file
+    # Choices are volume|plane|list|file|clear_and_add
```

Comment:
what do you think of "empty_and_fill", "empty_and_refill", or "dump_and_fill"/"dump_and_refill"?

Reply:
I would suggest remove_and_insert or erase_and_insert
Clearing gives us the idea of clearing the memory of a data structure. In essence, you are really removing particles.
I really prefer to use the words "remove" and "insert" because these also correspond to the words used in the ParticleHandler class.